service/matching/matchingEngine_test.go

```diff
@@ -195,7 +196,7 @@ func (s *matchingEngineSuite) TestPollForDecisionTasksEmptyResult() {
 
 func (s *matchingEngineSuite) PollForTasksEmptyResultTest(taskType int) {
 	s.matchingEngine.config.RangeSize = 2 // to test that range is not updated without tasks
-	s.matchingEngine.config.LongPollExpirationInterval = 10 * time.Millisecond
+	s.matchingEngine.config.LongPollExpirationInterval = func() time.Duration { return 10 * time.Millisecond }
```

Comment:
May be introduce small helper method? 
     s.matchingEngine.config.LongPollExpirationInterval  = DurationConfigMillis(10)